packages/orbit-components/src/Card/Card.stories.tsx

```diff
@@ -307,7 +309,7 @@ export const CardWithMixedSections: Story = {
         expandable
         title={sectionTitle}
         actions={
-          <ButtonLink compact size="small" type="secondary">
+          <ButtonLink onClick={buttonClick} compact size="small" type="secondary">
```

Comment:
I added this `action('buttonClick')` to be able to see that click action is working in SB.